### ISSUE TITLE
More careful image inspection on upload

### ISSUE THREAD

**Issue by beijbom** [beijbom/coralnet#115]
There are a few images on the site that tripped up the backend service (I discovered these / because they were part of ~20 images that didn't make it through the vision backend...)

They seem to come from the various BREAM source, notably: https://coralnet.ucsd.edu/source/683/
Here are two examples:
https://coralnet.ucsd.edu/image/545661/view/
https://coralnet.ucsd.edu/image/555867/view/

Looking closer; loading them in PIL works, but on converting them to numpy gives an empty array, so the backend crashed. I did find an (absurd) workaround on spacer, like so:

im_pil = Image.open('imagefile.jpg')
im = np.array(im_pil)
im = np.array(im_pil)

In other words, calling np.array 2 times actually does the trick (!). 

Clearly, this also tripps up the thumbnail creation code on the site, b/c the thumbnails are all strange.

I think the error is similar to what is described here:
`https://github.com/scikit-image/scikit-image/issues/2154`

but either way, the best case scenario would be to catch these problem on upload, and ask the user to provide a better copy of the image.

**Comment by beijbom** [comment]
Closing this. I'm pretty sure it was resolved somewhere along the way.
